src/dashboard/src/templates/transfer/detail.html

```diff
@@ -18,14 +18,14 @@
         <div class="alert-message block-message">
           <p>There are micro-services awaiting user decisions.</p>
           <div class="alert-actions">
-            <a class="btn small" href="{% url 'components.ingest.views.ingest_microservices' uuid %}">View micro-services</a>
```

Comment:
It looks like this also fixes a bug where a transfer detail used to incorrectly link back to an ingest's list of micro-services. Is that right?